.github/workflows/generate.yml

```diff
@@ -0,0 +1,58 @@
+name: Update and Create PR
+
+on:
+  schedule:
+    - cron: '0 0 1 * *' # Runs on the first day of each month at midnight
+  workflow_dispatch: # Allows the workflow to be triggered manually
+
+jobs:
+  update-and-pr:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository 🛎️
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js 🛠️
+        uses: actions/setup-node@v2
+        with:
+          node-version: 'lts/*' # Use the LTS version of Node.js
+
+      - name: Install dependencies 📦
+        run: npm install
+
+      - name: Run update script 🔄
+        run: npm run update
+
+      - name: Check for changes and create branch 📤
+        id: check_changes
+        run: |
+          git config --global user.name "github-actions[bot]"
+          git config --global user.email "github-actions[bot]@users.noreply.github.com"
+          if [ -n "$(git status --porcelain)" ]; then
+            BRANCH_NAME="update-branch"
+            if git show-ref --verify --quiet "refs/heads/$BRANCH_NAME"; then
+              TIMESTAMP=$(date +%s)
+              BRANCH_NAME="${BRANCH_NAME}-${TIMESTAMP}"
+            fi
+            git checkout -b $BRANCH_NAME
+            git add .
```

Comment:
not the most elegant thing to do here, since this should only ever update `db.json` and `src/`

Thoughts?

Reply:
Made this change